app/assets/stylesheets/finder_frontend.scss

```diff
@@ -324,7 +316,8 @@
 }
 
 .result-info {
-  margin-bottom: govuk-spacing(2);
+  margin: 0 0 govuk-spacing(4) 0;
```

Comment:
Did this override the previous version because it was defined in 
```
#finder-frontend { 
  .filtered-results {	
    .result-info {	
      margin: 0 0 govuk-spacing(4) 0;
}}} ```?

Reply:
Yeah - turns out there were two declarations for `.result-info`. The one with the `margin` declaration was overriding the one with the `margin-bottom` declaration because it was nested under the ID (the other one wasn't). So I replaced that one with this one.